elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -294,6 +294,32 @@
             .setRestartAllServices()
             .build();
 
+    static final SimpleAttributeDefinition RESPONDER = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.RESPONDER, ModelType.STRING, true)
+            .setAllowExpression(true)
+            .setRestartAllServices()
+            .build();
+
+    static final SimpleAttributeDefinition PREFER_CRLS = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.PREFER_CRLS, ModelType.BOOLEAN, true)
+            .setRestartAllServices()
+            .build();
+
+    static final SimpleAttributeDefinition SOFT_FAIL = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.SOFT_FAIL, ModelType.BOOLEAN, true)
+            .setDefaultValue(new ModelNode(false))
+            .setRequired(false)
+            .setRestartAllServices()
+            .build();
+
+    static final SimpleAttributeDefinition ONLY_LEAF_CERT = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.ONLY_LEAF_CERT, ModelType.BOOLEAN, true)
+            .setDefaultValue(new ModelNode(false))
+            .setRequired(false)
+            .setRestartAllServices()
+            .build();
+
+    static final ObjectTypeAttributeDefinition OCSP = new ObjectTypeAttributeDefinition.Builder(ElytronDescriptionConstants.OCSP, RESPONDER, PREFER_CRLS)
+            .setRequired(false)
+            .setRestartAllServices()
```

Comment:
I am a bit confused, as this comment is on OCSP and the XSD references prefer-crls, so I assume you are talking about that. The default is in the code that resolves the attribute value, but I guess it would be better in the attribute definition as you suggested

Reply:
Oh, sorry, my bad - I commented on wrong line. Anyway you decoded that correctly - yes, I was talking about `prefer-crls` attribute. The thing is that specifying default in attribute definition itself will be propagated to the attribute description which is visible to user when using a CLI. As such default value is clearer to user (it not only hidden in the code).